src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1001,6 +1001,18 @@
      "CREATE INDEX idx_resource_events_timestamp ON resource_events(timestamp)"
      "ALTER TABLE resource_events ADD CONSTRAINT resource_events_report_id_fkey FOREIGN KEY (report_id) REFERENCES reports(id) ON DELETE CASCADE"))
 
+(defn insert-report-metrics-and-logs
```

Comment:
Seems like it would be really easy to collapse into the previous migration, or are you worried about conflict?